lib/decanter/core.rb

```diff
@@ -50,12 +54,15 @@ def ignore(*args)
       end
 
       def strict(mode)
-        raise(ArgumentError, "#{self.name}: Unknown strict value #{mode}") unless [:ignore, true, false].include? mode
+        raise(ArgumentError, "#{name}: Unknown strict value #{mode}") unless [:ignore, true, false].include? mode
+
         @strict_mode = mode
       end
 
       def log_unhandled_keys(mode)
-        raise(ArgumentError, "#{self.name}: Unknown log_unhandled_keys value #{mode}") unless [true, false].include? mode
+        raise(ArgumentError, "#{name}: Unknown log_unhandled_keys value #{mode}") unless [true,
+                                                                                          false].include? mode
```

Comment:
This is an interesting change. I haven't seen formatting like this before. Did this happen as a result of some local formatting tool that you have running on save?
![Image 2024-05-24 at 4 19 22 PM](https://github.com/LaunchPadLab/decanter/assets/20426095/ff3ecc24-4752-4772-ad3c-92881921020c)